scripts/zones/The_Garden_of_RuHmet/mobs/Jailer_of_Fortitude.lua

```diff
@@ -11,18 +11,11 @@ require("scripts/globals/magic");
 -----------------------------------
 
 function onMobSpawn(mob)
-	-- Give it two hour
-	mob:setMod(MOBMOD_MAIN_2HOUR, 1);
-	-- Change animation to humonoid
-	mob:AnimationSub(1);
-	-- Set the damage resists
-	-- According to https://www.bg-wiki.com/bg/Jailer_of_Fortitude
-	-- Has very high resistance to melee damage (approximately 95%).
-	-- Set damage resistance to match that.
-	mob:setMod(MOD_HTHRES,50);
-	mob:setMod(MOD_SLASHRES,50);
-	mob:setMod(MOD_PIERCERES,50);
-    mob:setMod(MOD_IMPACTRES,50);
+    -- Give it two hour
+    mob:setMod(MOBMOD_MAIN_2HOUR, 1);
```

Comment:
i see that this is from the script before, but you can't use MOBMOD in setMod, you must use setMobMod (currently this add 1 MOD_ICEDEF)